src/helpers.ts

```
@@ -251,6 +251,14 @@ export function getNapiNativeTarget(): string[] | string {
 
       break
     }
+    case 'aix':{
+      return 'aix-ppc64'
+      break
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)